lunatrace/bsl/frontend/src/pages/homepage/AuthenticatedHome.tsx

```diff
@@ -24,7 +25,7 @@ import { SetupWizard } from './cards/SetupWizard';
 
 export const AuthenticatedHome: React.FunctionComponent = (_props) => {
   const wizardOpen = useContext(WizardOpenContext);
-
+  const [trigger, result, lastPromiseInfo] = api.useLazyGetSidebarInfoQuery();
```

Comment:
Not a nit -- just curious -- do you need to assign out these variables? I don't see any usages of them so I'm curious why you did (and how the linter allows this?)